core/src/main/resources/error/error-classes.json

```diff
@@ -48,13 +48,13 @@
   "FAILED_EXECUTE_UDF" : {
     "message" : [ "Failed to execute user defined function (<functionName>: (<signature>) => <result>)" ]
   },
+  "FAILED_PERMISSION_RESET_ORIGINAL" : {
```

Comment:
How about `FAILED_TO_SET_ORIGINAL_PERMISSION_BACK`?
I didn't follow the error message improvement project closely. Do we have a naming pattern?

Reply:
> Do we have a naming pattern?

no, we don't as far as I know.

Reply:
> How about FAILED_TO_SET_ORIGINAL_PERMISSION_BACK?

I would be nice to have shorter names for error classes since we are going to use them as tags in docs headers.

Reply:
SET_PERMISSION_BACK is shorter than RESET_PERMISSION_TO_ORIGINAL ;-)